src/vs/base/browser/ui/tree/compressedObjectTreeModel.ts

```diff
@@ -49,11 +49,11 @@ export function compress<T>(element: ICompressedTreeElement<T>): ITreeElement<IC
 			break;
 		}
 
-		element = children[0];
-
-		if (element.incompressible) {
+		let childElement: ICompressedTreeElement<T> = children[0];
+		if (childElement.incompressible) {
 			break;
 		}
+		element = childElement;
```

Comment:
I was getting an incorrect behavior when an ISCMResourceGroup contained just one element. This changes fix that.
![scmrememberstatebug](https://user-images.githubusercontent.com/25115070/73149813-0b2aa080-4091-11ea-859b-0475e337d264.gif)